ODIN_II/regression_test/benchmark/task/syntax/synthesis_result.json

```diff
@@ -804,90 +804,70 @@
         "verilog": "fir_scu_rtl_restructured_for_cmm_exp.v",
         "exit": 134,
         "errors": [
-            "PARSE_TO_AST Parser found (2) errors in your syntax, exiting"
+            "[PARSE_TO_AST] Parser found (2) errors in your syntax, exiting"
         ],
         "warnings": [
-            "PARSE_TO_AST fir_scu_rtl_restructured_for_cmm_exp.v:94 error in parsing: (syntax error, unexpected vSYMBOL_ID, expecting voLTE or '=')",
-            "PARSE_TO_AST fir_scu_rtl_restructured_for_cmm_exp.v:109 error in parsing: (syntax error, unexpected voLTE, expecting vSYMBOL_ID or '#')"
+            "fir_scu_rtl_restructured_for_cmm_exp.v:94:1 [PARSE_TO_AST] error in parsing: (syntax error, unexpected vSYMBOL_ID, expecting voLTE or '=')",
+            "fir_scu_rtl_restructured_for_cmm_exp.v:109:8 [PARSE_TO_AST] error in parsing: (syntax error, unexpected voLTE, expecting vSYMBOL_ID or '#')"
         ]
     },
     "syntax/fir_scu_rtl_restructured_for_cmm_exp/no_arch": {
         "test_name": "syntax/fir_scu_rtl_restructured_for_cmm_exp/no_arch",
         "verilog": "fir_scu_rtl_restructured_for_cmm_exp.v",
         "exit": 134,
         "errors": [
-            "PARSE_TO_AST Parser found (2) errors in your syntax, exiting"
+            "[PARSE_TO_AST] Parser found (2) errors in your syntax, exiting"
         ],
         "warnings": [
-            "PARSE_TO_AST fir_scu_rtl_restructured_for_cmm_exp.v:94 error in parsing: (syntax error, unexpected vSYMBOL_ID, expecting voLTE or '=')",
-            "PARSE_TO_AST fir_scu_rtl_restructured_for_cmm_exp.v:109 error in parsing: (syntax error, unexpected voLTE, expecting vSYMBOL_ID or '#')"
+            "fir_scu_rtl_restructured_for_cmm_exp.v:94:1 [PARSE_TO_AST] error in parsing: (syntax error, unexpected vSYMBOL_ID, expecting voLTE or '=')",
+            "fir_scu_rtl_restructured_for_cmm_exp.v:109:8 [PARSE_TO_AST] error in parsing: (syntax error, unexpected voLTE, expecting vSYMBOL_ID or '#')"
         ]
     },
     "syntax/flip_flop_enable/k6_frac_N10_frac_chain_mem32K_40nm": {
         "test_name": "syntax/flip_flop_enable/k6_frac_N10_frac_chain_mem32K_40nm",
         "architecture": "k6_frac_N10_frac_chain_mem32K_40nm.xml",
         "verilog": "flip_flop_enable.v",
+        "exit": 134,
```

Comment:
Why is this exiting now when it didn't before?

Reply:
good catch :+1: 
the error message was going out of bounds.